documentation/design-docs/ipc-protocol.md

```diff
@@ -416,15 +435,36 @@ Payload
 ```
 Followed by an Optional Continuation of a `netperf` format stream of events.
 
-The `StopTracing` Command consumes the following Payload:
+### `StopTracing` 
+
+Command Code: `0x0201`
+
+The `StopTracing` command is used to stop a specific streaming session.  Clients are expected to use this command to stop streaming sessions started with [`CollectStreaming`](#CollectStreaming).
+
+#### Inputs:
+
+Header: `{ Magic; 28; 0x0201; 0x0000 }`
+
+* `ulong sessionId`: The ID for the streaming session to stop
+
+#### Returns:
+
+Header: `{ Magic; 28; 0x0201; 0x0000 }`
```

Comment:
Should this be 0xFF00 instead of 0x201?

Reply:
Yes, it should.

Reply:
On MacOS, it needs to be written in the application group container.  See this code in the PAL for details: https://github.com/dotnet/coreclr/blob/c03f31fbc4105cbea4053409f91bc25040093d20/src/pal/src/thread/process.cpp#L2336

I don't think we can always use $TMPDIR.  I think we need to follow how the debugger transport pipe names are built.  I don't know all the details; it was @odhanson that did all the application group work.